crates/storage/provider/src/trie/mod.rs

```diff
@@ -667,4 +819,44 @@ mod tests {
             test_with_accounts(accounts);
         });
     }
+
+    #[test]
+    fn get_proof() {
```

Comment:
let's please add a test for something that has multiple storage slots as well

Reply:
Added